quickfort.lua

```diff
@@ -102,16 +103,12 @@ configuration stored in the file:
     normally *decreases* the efficiency of your fort.
 
 There is one other configuration file in the ``dfhack-config/quickfort`` folder:
-`aliases.txt`_. It defines keycode shortcuts for query blueprints. The format
-for this file is described in the file itself, and default aliases that all
-players can use and build on is stored in
-`hack/data/quickfort/aliases-common.txt`_.
+:source:`aliases.txt <dfhack-config/quickfort/aliases.txt>`. It defines keycode
+shortcuts for query blueprints. The format for this file is described in the
+file itself, and default aliases that all players can use and build on is stored in
+:source:`hack/data/quickfort/aliases-common.txt <data/quickfort/aliases-common.txt>`.
 
-.. _blueprint plugin: https://docs.dfhack.org/en/stable/docs/Plugins.html#blueprint
-.. _Blueprints Guidebook: https://github.com/DFHack/dfhack/tree/develop/data/blueprints
 .. _blueprints/library: https://github.com/DFHack/dfhack/tree/develop/data/blueprints/library
```

Comment:
I think all references to this link have been converted and this line can be deleted

Reply:
Oh, it's failing because it's using the develop branch of DFHack, and https://github.com/DFHack/dfhack/pull/1661 isn't merged yet. It should pass once that PR is merged. But I should take care of the rest of these links, yes.